book/src/04_4_1_interrupts.md

```diff
@@ -19,11 +19,18 @@ cargo run --example solution
     - See `hal::task::notification` documentation
 3. In an `unsafe` block, create a subscription and its callback function.
     - See `PinDriver::subscribe` and `task::notify_and_yield`
+    - The reasons for being `unsafe` are:
+      - The callback function will be the [ISR (Interrupt Service Routine)](https://en.wikipedia.org/wiki/Interrupt_handler) itself, so we should avoid calling any blocking functions on it, this includes STD, `libc` or FreeRTOS APIs (except for a few allowed ones).
```

Comment:
not will be - better run in it, as its not technically the full ISR

Reply:
Updated!